validate_release.py

```diff
@@ -318,6 +342,8 @@ def ParseDeps(dpk, out):
 
 def AnalyzeDpk(dpk, unv, symids, deps):
     dpk = zipfile.ZipFile(dpk)
+    if 'dirty' in dpk.filename:
+        yield 'Dirty version string detected: ' + dpk.filename
```

Comment:
You can just say “Dirty package detected”.

This version string happens when something is not committed when building the package. I had a stray `.orig` from a merge conflict in the dpkdir.

I plan to blacklist those `.orig` filesin urcheon, but there are more scenarii where stray files can occurs…